Sources/Errors.php

```diff
@@ -79,14 +79,17 @@ function log_error($error_message, $error_type = 'general', $file = null, $line
 		$user_info['ip'] = '';
 
 	// Find the best query string we can...
-	$query_string = empty($_SERVER['QUERY_STRING']) ? (empty($_SERVER['REQUEST_URL']) ? '' : str_replace($scripturl, '', $_SERVER['REQUEST_URL'])) : $_SERVER['QUERY_STRING'];
+	if (SMF == 'SSI')
```

Comment:
```suggestion
	if (SMF === 'SSI')
```
perf:  Identity check is slightly faster